ci-operator/config/openshift-priv/ovn-kubernetes/openshift-priv-ovn-kubernetes-master.yaml

```diff
@@ -416,14 +416,13 @@ tests:
       COMPUTE_NODE_TYPE: m5.xlarge
       FEATURE: NetworkSegmentation
       ITERATION_MULTIPLIER_ENV: "3"
-      KUBE_BURNER_VERSION: 1.5.0
     post:
     - chain: ipi-aws-post
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
     - chain: openshift-qe-installer-featuregate
-    workflow: openshift-qe-udn-density-l2-pods
+    workflow: openshift-qe-udn-density-pods
```

Comment:
Doesn't this job need to set `ENABLE_LAYER_3=false` now? (and likewise all the other -l2 jobs).

This change seems like the sort of thing that is likely to lead to jobs accidentally being defined incorrectly, such that that "-l3" and "-l2" versions of a job both end up testing L3.

Reply:
I think this might be done in the next commit?

Reply:
Updated, ptal